app/javascript/lib/UseCases/Rondels/MoveToSlot.js

```diff
@@ -0,0 +1,62 @@
+import AvailableSlots from './SlotSelection/AvailableSlots';
+import SlotDistanceCosts from './SlotSelection/SlotDistanceCosts';
+
+const InvalidMoveError = class extends Error {
+  constructor(fromRondelSlot, toRondelSlot) {
+    super(`Attepting to make invalid move from ${fromRondelSlot?.id} to ${toRondelSlot.id}.`);
+
+    this.name = 'InvalidMoveError';
+  }
+};
+
+export default class MoveToSlot {
+  static get InvalidMoveError() {
+    return InvalidMoveError;
```

Comment:
Im inclined to disagree and ignore lint's recommendation of `Only 1 class per file`

With this setup, we can write nested classes, which indicates a high degree of relationship between the 2 classes. As an external user, its access would look like
```
MoveToSlot.InvalidMoveError
```
as opposed to
```
import InvalidMoveError from './InvalidMoveError'
```
which doesn't really hint to me that its relationship with others modules

Reply:
Would like to hear your thoughts before doing anything